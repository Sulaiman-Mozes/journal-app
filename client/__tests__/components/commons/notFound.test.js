import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import NotFound from '../../../components/commons/notFound';

describe('Provider and NotFound', () => {
  const notFound = shallow(<NotFound />);

  it('renders <Provider/> correctly', () => {
    expect(notFound).toMatchSnapshot();
  });
});
