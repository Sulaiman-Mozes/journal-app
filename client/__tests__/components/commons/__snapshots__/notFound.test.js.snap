// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and NotFound renders <Provider/> correctly 1`] = `
<div
  className="flex-notfound text-center"
>
  <div>
    <h2>
      Something went wrong
    </h2>
    <br />
    <p>
      Sorry, the page you are requesting for was not found
    </p>
  </div>
</div>
`;
